haralyzer/http.py

```diff
@@ -250,9 +270,17 @@ def mimeType(self) -> str:
         return self.raw_entry["content"]["mimeType"]
 
     @cached_property
-    def text(self) -> str:
+    def text(self) -> Optional[str]:
         """
         :return: Response body
         :rtype: str
         """
-        return self.raw_entry["content"]["text"]
+        return self.raw_entry["content"].get("text")

```

Comment:
response.content is required but response.content.text is optional. so, I fixed it too this time.
however, this could appear **breaking** change for users. the actual impact is shown in test_chrome_entry.py::test_response.

Reply:
Is there a reason that this change needs to be made with this PR?

Reply:
no special reason. will revert it.